src/cmarkConfig.cmake.in

```diff
@@ -3,5 +3,5 @@ set(HAVE_LIBCMARK_SHARED @CMARK_SHARED@)
 
 if((HAVE_LIBCMARK_STATIC AND NOT TARGET libcmark_static) OR
    (HAVE_LIBCMARK_SHARED AND NOT TARGET libcmark))
-  include(cmarkTargets.cmake)
+ include(${CMAKE_CURRENT_LIST_DIR}/cmarkTargets.cmake)
```

Comment:
Is the indentation change intentional?

Reply:
No, the indentation change wasn't intentional.